cfgov/v1/tests/test_wagtail_hooks.py

```diff
@@ -192,3 +193,48 @@ def test_get_resource_tags_returns_alphabetized_list(self):
             get_resource_tags(),
             [(u'atag', u'aTag'), (u'taga', u'tagA'), (u'tagb', u'tagB')]
         )
+
+
+class TestResourceTagsFilter(TestCase, WagtailTestUtils):
+    def setUp(self):
+        self.login()
+        self.resource1 = Resource.objects.create(title='Test resource Orange')
+        self.resource1.tags.add(u'tagA')
+        self.resource2 = Resource.objects.create(title='Test resource Banana')
+        self.resource2.tags.add(u'tagB')
+        self.resource3 = Resource.objects.create(title='Test resource Apple')
+        self.resource3.tags.add(u'tagB')
+        self.resource3.tags.add(u'tagC')
+
+    def get(self, **params):
+        return self.client.get('/admin/v1/resource/', params)
+
+    def test_no_params_returns_all_resources_in_order(self):
+        response = self.get()
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.context['result_count'], 3)
+        self.assertEqual(
+            list(response.context['object_list']),
+            list(Resource.objects.all().order_by('title'))
```

Comment:
Nit:
- `.order_by('title')` seems unnecessary here, because the model specifies the sort. The test succeeds without it.

Reply:
Huh! That's surprising, because the [ordering on the model is actually `'order', 'title'`](https://github.com/cfpb/cfgov-refresh/pull/4529/files#diff-1a53f67ccb2f8350dd16136cfe0378d2R110). But you're right, it passes without that.

Reply:
I just realized that it's because I didn't set an `order` on any of the test snippets 🤦‍♂️ Update coming.

Reply:
Update pushed (amended previous commit and rebased everything on top of master).